package.json

```diff
@@ -37,7 +39,6 @@
     "typescript": "^3.8.0"
   },
   "peerDependencies": {
-    "aws-sdk": "^2.841.0",
     "ion-js": "^4.0.0",
     "jsbi": "^3.1.1"
   },
```

Comment:
This PR should probably also bump our local major version, or at least a minor version if the exposed API has not changed.